### PR TITLE
Add support for verifying comm-central based decision tasks.

### PR DESCRIPTION
**Pull request by tomprince**
Depends on #170.

### CODE REVIEW

**Comment by escapewindow** [comment]
@tomprince is this ready for review? If so, could you unbitrot?

**Comment by tomprince** [comment]
This can probably be reviewed, but it would be better to test this in a branched environment before merging, probably.

**Comment by escapewindow** [comment]
Ok. Maybe that's something @JohanLorenzo can help with by setting up dev TB scriptworkers?

I'm mainly trying to avoid a bitrot collision with https://github.com/mozilla-releng/scriptworker/compare/master...escapewindow:cotv2?expand=1 , which isn't ready yet, but will be in the coming days. If this PR is ready before or at around the same time, I'm happy to let this land first.

File: scriptworker/constants.py [hunk 1]
```diff
@@ -186,21 +186,34 @@
...
-            #     migration is tier1 and landed on mozilla-central
-            # XXX remove /projects/jamun when we no longer run staging releases
-            #     from it
-            # XXX remove /projects/oak when we no longer test updates against it
-            "^(?P<path>/projects/(birch|maple|jamun|oak))(/|$)",
-        ),
-    }), ),
+    'valid_vcs_rules': {
+        'by-cot-product': frozendict({
+            'firefox': frozendict({
```
Comment (escapewindow) [inline scriptworker/constants.py]: This should be a tuple of frozendicts; that's probably why tests are failing.

**Comment by JohanLorenzo** [comment]
Sorry, I hit [some permissions issues](https://bugzilla.mozilla.org/show_bug.cgi?id=1424254#c9) on Wednesday which held the set up of the machines. I'll resume it today.

**Comment by JohanLorenzo** [comment]
I'm glad to tell you CoT successfully ran with this PR applied at https://queue.taskcluster.net/v1/task/Unwpf7TwTiGXonRbXEWIIA/runs/8/artifacts/public/logs/chain_of_trust.log

**Comment by JohanLorenzo** [comment]
@tomprince Could you rebase this PR? Once rebased, I'll make sure signing is not broken, before we merge it to master. What do you think?

**Comment by tomprince** [comment]
I've updated this, and verified some windows signing tasks with the new code.

**Comment by escapewindow** [comment]
Go for it! I think we have to update the scriptworker.yaml templates in puppet when we roll out.

**Comment by JohanLorenzo** [comment]
> Go for it! I think we have to update the scriptworker.yaml templates in puppet when we roll out.

Done in https://bugzilla.mozilla.org/show_bug.cgi?id=1421062#c26

**Pull request merged**